spec/raven/instance_spec.rb

```diff
@@ -245,6 +245,40 @@ def ivars(object)
     end
   end
 
+  describe "#user_context" do
+    context "without a block" do
+      it "empties the user context when called" do
```

Comment:
nitpick: think this should be `"empties the user context when called without options"`